### PR TITLE
Add Turku workshop

### PR DESCRIPTION
**Pull request by pwalczysko**
Upcoming workshop added.

Built at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/

### CODE REVIEW

File: events/omero-users-workshop-turku-online-november-2021.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="column">
+                <h2 class="event-day">Duration: 3 hours per day</h2>
+                <p class="event-day-description">The Open Microscopy Environment (OME) is an open-source software project that develops tools that enable access, analysis, visualization, sharing and publication of biological image data. We support more than 150 image data formats across many imaging modalities including medical imaging, high-content screening and whole-slide imaging.
+                We support and visualize multi-channel 3D timelapse images.
+ 
+                OMERO, a component of OME, is a software platform for image data management and analysis which was presented in this course.
+                Import, organisation, viewing, searching, annotation, analysis with Fiji and publishing of imaging data with OMERO was shown.
+                </p>
+                <p class="event-day-description">
+                The workshop concentrated on basics of OMERO and did not assume any programming skills.
```
Comment (joshmoore) [inline events/omero-users-workshop-turku-online-november-2021.html:34]: This is in the past-tense but I assume since it's within a week that's fine.

**Review by joshmoore — COMMENTED** [review]
One minor thought which I leave to you, Petr, whether there's any synchronization to be done. (Do all pages stay in one tense? Seems like swapping would be a pain)

Otherwise, LGTM.

File: events/omero-users-workshop-turku-online-november-2021.html (hunk 1 above)
Comment (pwalczysko) [reply events/omero-users-workshop-turku-online-november-2021.html:34]: If we wait till tomorrow, it will become a past tense :)

**Comment by pwalczysko** [comment]
I think I could have a look later in the week about how to synchronize.

**Comment by pwalczysko** [comment]
@joshmoore after some searching and comparing, I would suggest to drop any sync attempt on tenses. The senstences do not vary too much, but slightly for sure. Expression such as ``will cover``, ``will show``, ``will be shown``, ``will demonstrate``  are very variable. Some future tense is also in licence files. The danger when, say, replacing ALL ``will cover`` with ``was covered`` is that in some of the paragraphs the future tense will remain in the next sentence, making the context even more confusing.

**Comment by pwalczysko** [comment]
The https://github.com/ome/www.openmicroscopy.org/pull/546/commits/3ecd47233c34d4fec49c33fc402d9dc6fbe8cf92 should synchronize all 2021 workshops to past  tense.

**Comment by pwalczysko** [comment]
Synchronized the tenses in past three years @joshmoore - I would argue that the years before that, the workshops were a bit different, and older than 2019 materials and events will hardly be revisited.

File: events/omero-dev-workshop-dundee-september-2019.html [hunk 2]
```diff
@@ -41,18 +41,18 @@ <h2 class="event-day">Duration: 2 days workshops</h2>
...
                     <li>integrating analysis tools</li>
                 </ul>	
                 </p>
                 <p>
-                Additionally, we would like to continue the discussion from last year's users meeting and the <a href="https://forum.image.sc/t/proposal-for-4d-nucleome-microscopy-metadata-guidelines-based-on-the-ome-data-model/25970">more recent</a> community zoom call on building a framework for capturing, managing and querying metadata outside the OME-XML model.
+                Additionally, we continued the discussion from last year's users meeting and the <a href="https://forum.image.sc/t/proposal-for-4d-nucleome-microscopy-metadata-guidelines-based-on-the-ome-data-model/25970">more recent</a> community zoom call on building a framework for capturing, managing and querying metadata outside the OME-XML model.
                 <p>
                 <p>
-                If there are other topics that you would like to suggest, please free to list them on your registration form.
+                Paricipansts were encouraged to list other potential topics on the registration forms.
```
Comment (jburel) [inline events/omero-dev-workshop-dundee-september-2019.html:55]:
```suggestion
                Participants were encouraged to list other potential topics on the registration forms.
```

Comment (pwalczysko) [reply events/omero-dev-workshop-dundee-september-2019.html:55]: ta

**Comment by joshmoore** [comment]
> Synchronized the tenses in past three years @joshmoore - I would argue that the years before that, the workshops were a bit different, and older than 2019 materials and events will hardly be revisited.

:+1: (though that was certainly above & beyond the call). So in the future: what's the plan? Present tense before and then swap to past? Or use the same tense for before & after?

**Comment by pwalczysko** [comment]
> So in the future: what's the plan? Present tense before and then swap to past? Or use the same tense for before & after?

I would say, when the post is done very near the event or after the event, past tense. If the post is done well in advance, then we give it a future tense, but then later, when we come to new posts, we will swap the tense as we are now also updating the posts with the links to the "materials", which are always done only very close to the event itself, just like I have just done in https://github.com/ome/www.openmicroscopy.org/pull/546/commits/cf6e1e824954f298b3a065804b9b7c30783506a6

**Comment by joshmoore** [comment]
Understood & WFM. Thanks. :+1:

**Comment by pwalczysko** [comment]
@jburel Just pushed a new commit. All is built at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/

**Comment by sbesson** [comment]
Is this ready to merge?

**Comment by jburel** [comment]
@pwalczysko I made a few adjustments: 
- removed whitespace and tabs
- fixed space between words in i2k workshop.

if we are happy with those adjustments, we should be ready to go

**Comment by pwalczysko** [comment]
> @pwalczysko I made a few adjustments:
> 
>     * removed whitespace and tabs
> 
>     * fixed space between words in i2k workshop.
> 
> 
> if we are happy with those adjustments, we should be ready to go

@jburel thank you very much for that. I have built again and checked at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ and all looks good.

@jburel Do you want to press the merge button ?

**Pull request merged**